server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -13,7 +13,10 @@ export default class ServiceProviderReferralsController {
   ) {}
 
   async showDashboard(req: Request, res: Response): Promise<void> {
-    const referrals = await this.interventionsService.getSentReferrals(res.locals.user.token)
+    const referrals = await this.interventionsService.getSentReferrals(
+      res.locals.user.token,
+      res.locals.user.organizations[0].code
```

Comment:
i think this raises a question about what we do with this list of orgs..

@sldblog are all the questions about primes/subs answered to the point we can make some decisions about org level access now?

Reply:
Currently, we can assume
- one provider per user
- more than one provider: 🚫 invalid state

After onboarding subcontractors (let's not worry about this now, though)
- non-admin users: one organisation per user
- admin users: many organisations per user
- admin users: one _prime_ organisation per user, more than one prime: 🚫 invalid state

Does this help?

Reply:
For now, it would be good to explicitly flop the login if we have more than 1 org than rely on potential ordering chaos 😅

Reply:
However, longer-term:

- each user will have `0..*` contracts assigned to them
- we will need to get the sent referrals only for those specific contract(s)
- intersected by their providers

Reply:
Is it possible to completely defer the scoping to the API based on the user token?

Reply:
done. closing this pR

Reply:
https://github.com/ministryofjustice/hmpps-interventions-service/pull/107